.golangci.yml

```diff
@@ -95,6 +95,7 @@ linters-settings:
 linters:
   disable:
     - ireturn
+    - wsl
```

Comment:
please remove the changes to the linters

Reply:
done.